adoc/code/queueShortcuts.cpp

```diff
@@ -7,7 +7,7 @@ queue myQueue;
 auto usmPtr = malloc_device<int>(1024, myQueue);  // USM pointer
 
 int* data = /* pointer to some data */;
-buffer buf{data, 1024};
+buffer<int, 1> buf{data, range<1>{1024}};
```

Comment:
```suggestion
buffer buf{data, {1024}};
```

I think this is all you need.  (I checked on DPC++, and this does compile.)

The type `int` is deduced via CTAD from that `data` parameter and the dimension `1` is deduced from the range.  We just needed to add curlys around the `1024`, so that it list-initializes a `sycl::range`.

We added all the CTAD stuff to SYCL 2020 to make code less verbose, and I think our examples should make use of it.

Reply:
> We added all the CTAD stuff to SYCL 2020 to make code less verbose, and I think our examples should make use of it.

Thanks @gmlueck ! Should we align other examples to make it consistent?

Reply:
I haven't looked at all the examples to see if they could be improved, but I agree that consistency is good.  If you know of examples that could be made better, feel free to add to this PR or creates others.

Reply:
I've fixed the obvious issues now with this PR. I'd suggest doing a follow up PR for alignment.
Alignment of examples is a lot and might require some discussion.